{
    "log_id": "log_093",
    "timestamp": "2025-06-09T19:25:49.014002",
    "event_type": "user_login_attempt",
    "user_id": "user_139",
    "details": {
        "ip_address": "192.168.1.11",
        "reason": "Invalid credentials",
        "success": false
    }
}